modules/experimental/pages/robust.adoc

```diff
@@ -0,0 +1,102 @@
+= Robust
+
+`Unit ability`
+
+An experimental rule to allow some normal units to have Damage Capacity `2`.
+
+This is similar to how Damage Capacity works with War Engine units.
+That said...
+
+IMPORTANT: Units that have the {robust} ability are not War Engine units — do not apply the War Engine rules to them.
+
+In brief, it takes two hits to destroy a {robust} unit, and each {robust} unit counts as two normal units for both hit allocation and detachment strength.
+
+[NOTE]
+====
+We're testing this ability as a way to make some units tougher than the standard rules allow, but not as tough as War Engine units.
+Candidates for this ability include Space Marine Land Raiders, Ork Stompas, and Chaos Greater Daemons.
+====
+
+---
+
+Where your unit has the {robust} ability:
+
+* The unit starts the game with Damage Capacity `2`.
+* When you calculate the detachment strength, count the unit as equal to its remaining Damage Capacity.
+* Each hit reduces its Damage Capacity by one point.
+* If the unit's Damage Capacity is reduced to 0 then it is destroyed.
+
+Where you attack a detachment that includes units that have the {robust} ability:
+
+* Each {robust} unit counts as *two* units for hit allocation.
+This means that when you allocate hit dice and you reach a {robust} unit, you must allocate *two* dice to that unit before you allocate dice to more distant targets.
+// TODO: Do we still have to allocate two dice even if the unit has only 1 DC remaining?
+* Where possible, you must allocate two dice that score equal to or higher than the {robust} unit's Armour value, *or* two dice that score lower.
```

Comment:
> TODO: Do we still have to allocate two dice even if the unit has only 1 DC remaining?

Hmm. I'm not sure. It probably makes sense to change it to the number of remaining DC, but so far it's come up very rarely.

```suggestion
This means that when you allocate hit dice and you reach a {robust} unit, you must allocate as many dice to that unit as it has remaining Damage Capacity, before you allocate dice to more distant targets.
* Where possible, all dice allocated to the {robust} unit must score equal to or higher than its Armour value, *or* all dice allocated must score lower.
```

Reply:
Okay, allocate against current/remaining DC. That's my gut feeling too.
Your suggestion looks good, I'd just like to tweak it a bit — and we also need to integrate line #31, which isn't included in the selection for this conversation thread. So, I'll make a new suggestion on a new thread that includes that line.